### ISSUE TITLE
Weighting of labels

### ISSUE THREAD

**Issue by kmunve** [NVE/avalanche_ml#7]
Some ML-models allow to weight the labels, by providing a dict with weights. E.g see class_weight for [RandomForestClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html).

With the current setup the same weight-dict is applied to all labels making the code eventually crash.
Ideally it should be possible to define a model for each label individually.
